ESLint semi-colon repair throws an error (but still works)

`npm run lint:fix` is throwing an error.  It still seems to fix the semi-colons, but I thought I should log this anyways.

```sh
⇒  npm run lint:fix   

> react-redux-starter-kit@0.18.0 lint:fix /Users/terry/Desktop/Projects/react-redux-starter-kit
> eslint **/* --fix


/Users/terry/Desktop/Projects/react-redux-starter-kit/src/index.html
  1:2  error  Parsing error: Unexpected token

✖ 1 problem (1 error, 0 warnings)


npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "lint:fix"
npm ERR! node v5.2.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! react-redux-starter-kit@0.18.0 lint:fix: `eslint **/* --fix`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-redux-starter-kit@0.18.0 lint:fix script 'eslint **/* --fix'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-redux-starter-kit package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint **/* --fix
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-redux-starter-kit
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-redux-starter-kit
npm ERR! There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "compile": "node -r dotenv/config --harmony bin/compile",
-    "lint:fix": "eslint **/* --fix",
+    "lint:fix": "eslint **/*.js --fix",
     "start": "npm run dev",
     "dev": "node -r dotenv/config bin/server --hot",
     "dev:nw": "npm run dev -- --nw",

